steps/app-gallery-deploy/1.0.0/step.yml

```diff
@@ -0,0 +1,74 @@
+title: Deploy to Huawei App Gallery
+summary: |
+  Uploads APK to Huawei App Gallery
+description: |
+  This step is based on [Huawei Publishing API](https://developer.huawei.com/consumer/en/service/hms/catalog/AGCConnectAPI.html?page=hmssdk_appGalleryConnect_getstarted)
+website: https://github.com/FutureMind/bitrise-step-huawei-app-gallery-apk-distribution
+source_code_url: https://github.com/FutureMind/bitrise-step-huawei-app-gallery-apk-distribution
+support_url: https://github.com/FutureMind/bitrise-step-huawei-app-gallery-apk-distribution/issues
+published_at: 2020-01-20T12:06:13.398185+01:00
+source:
+  git: https://github.com/FutureMind/bitrise-step-huawei-app-gallery-apk-distribution.git
+  commit: b16418ac3e83b0601b36d48627629432bfb4e509
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+- xamarin
+- react-native
+- flutter
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: jq
+  apt_get:
+  - name: jq
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- apk_path: $BITRISE_APK_PATH
+  opts:
+    description: Path to generated APK file to deploy.
+    is_expand: true
+    is_required: true
+    title: APK path
+- huawei_filename: build_${ANDROID_VERSION_NAME}_${ANDROID_VERSION_CODE}.apk
```

Comment:
Only [Bitrise exposed env vars](https://devcenter.bitrise.io/builds/available-environment-variables/) should be used as a default value for step inputs.
What about: `build_${BITRISE_BUILD_NUMBER}.apk`?

Reply:
Yes, I think this will work as well